auslib/web/admin/views/scheduled_changes.py

```diff
@@ -64,17 +65,14 @@ def get(self):
             ret["scheduled_changes"].append(scheduled_change)
         return jsonify(ret)
 
-    def _post(self, form, transaction, changed_by):
-        if not form.validate():
-            self.log.warning("Bad input: %s", form.errors)
-            return Response(status=400, response=json.dumps(form.errors))
+    def _post(self, what, transaction, changed_by):
+        what.pop("csrf_token", None)
 
-        # Forms can normally be accessed as a dict through form.data,
-        # but because some of the Forms we end up using have a Field
-        # called "data", this gets overridden, so we need to construct
-        # a dict ourselves.
-        columns = {k: v.data for k, v in form._fields.iteritems()}
-        sc_id = self.sc_table.insert(changed_by, transaction, **columns)
+        # Validate 'when' is not in past
+        if what.get("when", None) and int(what.get("when")) < getMillisecondTimestamp():
```

Comment:
sc_when_validator if declared on a field will automatically validate the request and response bodies both. In case of response bodies , it will always return an error when attempting to fetch a S.C.(which was enacted) and its 'when' value (stored in DB) was in past.